src/addandremove.js

```diff
@@ -0,0 +1,113 @@
+const TaskContainer = document.querySelector('.list-Container');
+
+class CreatetodoList {
+  constructor(description, completed, id) {
+    this.description = description;
+    this.completed = completed;
+    this.id = id;
+  }
+
+  static displayTasks = (task, oneTwo, three) => {
+    const listItem = document.createElement('li');
+    listItem.id = task.id;
+    listItem.className = 'lists';
+    listItem.innerHTML = `
+          <div class="list-Container">
+          <input type="checkbox" name="" id="${task.id}" class="checkbox" ${oneTwo}>
+          <p class="${three}" id="${task.id}">${task.description}</p>
+          </div>
+          
+          <div class="trash">
+          
+          <i id="pen" class="fa-solid fa-pen"></i>
+          <i id="delete" class="fa-solid fa-trash-can"></i>
+          </div>
+          `;
+
+    TaskContainer.appendChild(listItem);
+  };
+
+  static loadFromLocalStorage() {
+    let Tasks;
```

Comment:
- [ ] The same happens here, please consider using `camelCase`! 🤓 🔧 

```suggestion
    let tasks;
```

Reply:
I have changed Tasks to tasks